drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/dmn/KogitoScenarioSimulationBuilder.java

```diff
@@ -308,8 +311,10 @@ private DMN12UnmarshallCallback getDMN12UnmarshallCallback(final ScenarioSimulat
     private Simulation getDMNSimulation(final FactModelTuple factModelTuple) {
```

Comment:
Could we add a test on this model to check factMapping and column value?

Reply:
@dupliaka Tests added.